### DISCUSSION TITLE
Amazing Math library written in REXX

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#384] [General]
[Rexx Math library](https://rosettacode.org/wiki/Mathematics.hel#) It contains functions and constants I have never heard of!

**Comment by rvjansen** [comment]
This https://github.com/RexxLA/rexx-repository/tree/master/ARB/standards/work-in-progress/math gives some context.

**Comment by rvjansen** [comment]
I replied when the coffee did not do its work yet, with no reading glasses on, while under the impression you saw the rexxmath directory in lib/rxmath - but I now clicked the link and this would be an excellent way to fill that directory, when done with decimal types. You also know Vladimir Zabrodsky's work? Also, that could be input to the new Rexx standard.

best regard,

René.
